### ISSUE TITLE
Extended tuning does not enable

### ISSUE THREAD

**Issue by timvand** [ArduPilot/MissionPlanner#1613]
#### Issue details
Extended tuning is NOT visible in the Configuration tab, while Layout is set to advance on Planner.

#### Version
All

#### Platform
[X] All

#### Airframe type
Multirotor and fixed-wing

#### Hardware type
PH1/PH2

#### Logs
https://prnt.sc/fzxbia

![170724_mp_v1 3 49_missingextendedtune](https://user-images.githubusercontent.com/13006982/28611427-9d4b3c9e-71a0-11e7-82f1-b78daf1d3eb6.png)

**Comment by meee1** [comment]
what firmware are you connected to? plane copter etc

**Comment by timvand** [comment]
Plane.

**Comment by meee1** [comment]
copter has a page called extended tuning. 
https://github.com/ArduPilot/MissionPlanner/blob/master/GCSViews/SoftwareConfig.cs#L57

plane has a page called Basic tuning.
https://github.com/ArduPilot/MissionPlanner/blob/master/GCSViews/SoftwareConfig.cs#L62

this has not changed in multiple versions. so I'm not seeing a problem/issue at all.

**Comment by timvand** [comment]
What would be your suggestion/recommendation to accomplish this configuration in Plane?
![170726_mp_extenedparmissue-plane](https://user-images.githubusercontent.com/13006982/28655624-ec7487e2-7251-11e7-9655-491feeb67e10.jpg)

Specifically the portion related to configuring CH7 OPT to Camera Trigger, on Plane FW?

**Comment by meee1** [comment]
that feature/option is not available on plane. never has been.

**Comment by timvand** [comment]
Understood.  So that feature is not available on Plane and thus the option is not configurable, or is there some other method to achieve the same result?

**Comment by meee1** [comment]
you can trigger the camera from the GCS (flightdata > right click map). but I'm not aware of a method to trigger via a transmitter option, other than having a servo directly connected to that rx channel (ie manual bypassing).

**Comment by timvand** [comment]
So, not possible in plane and possible in copter?  How can that be?

**Comment by micaheli** [comment]
I agree that this is a ridiculous limitation.  Not having the ability to assign functions to switches at all is absolutely assinine.   Especially when functions of airplanes and multirotors have a lot of overlap.

The fact that this has not been re-addressed since 2017 hurts my brain.
